Gruntfile.js

```diff
@@ -93,24 +93,24 @@ module.exports = function(grunt) {
                     }]
                 }
             },
-            manifestIcons: {
+            manifestJsonIcons: {
                 files: {
                     'icons.js': 'icons.js',
                 },
                 options: {
                     replacements: [{
                         pattern: /<svg aria-hidden="true" class="svg-icon icon/g,
-                        replacement: '- helper: '
+                        replacement: '\n  {\n\    "helper": "'
```

Comment:
100% untested, but this might point you in the right direction

```suggestion
                        pattern: /^(<svg aria-hidden="true" class="svg-icon icon(.+?) width=".*<\/svg>)$/,
                        replacement: '\n "$2": "$1",'
```